docs/specs/helpers.md

```diff
@@ -56,3 +56,52 @@ The anchor point is highlighted with an orange dot.
 {schema_string:helpers/marker/description}
 
 {schema_object:helpers/marker}
+
+<h2 id="mask">Mask</h2>
+
+{schema_string:helpers/mask/description}
+
+{schema_object:helpers/mask}
+
+Masks provide single-channel coverage information (alpha channel) that modulates the layer's
+content.
+
+When multiple masks are specified, they are combined (blended) into a single coverage buffer,
+in order, based on the [`mode`](../constants/#mask-mode) operator.
+
+Masks are specified in terms of a `Path` plus additional properties.  For a given mask path,
+the coverage $C_{path}$ is $1$ inside the path, $0$ outside the path, and possibly in the $[0..1]$
+range along the path edges (anti-aliasing).
+
+The coverage for a given `Mask` is
+
+$$C = \begin{cases}{lr}
+  C_{path} \cdot opacity, & \text{when } inv = false \\
+  C_{path}^{-1} \cdot opacity, & \text{when } inv = true \\
+\end{cases}$$
+
+and the cumulative coverage for all masks is
+
+$$C_{cumulative} = \prod_{k=1}^{n} C_k$$
+
+where the product operator is determined by [`mode`](../constants/#mask-mode).
+Then the final layer coverage (alpha channel) is
+
+$$C_{layer}\prime = C_{layer} \cdot C_{cumulative}$$
+
+<lottie-playground example="mask.json">
+    <title>Example</title>
+    <form>
+        <input type="range" min="0" max="100" value="100" title="Opacity"/>
+        <input type="range" min="-40" max="40" value="0" title="Expansion"/>
+        <input type="checkbox" title="Invert"/>
+    </form>
+    <json>lottie.layers[1].masksProperties[0]</json>
+    <script>
+        let mask = lottie.layers[1].masksProperties[0];
+        mask.o.k = Number(data["Opacity"]);
+        mask.x.k = Number(data["Expansion"]);
+        mask.inv = data["Invert"];
```

Comment:
It looks like invert doesn't work properly in LottieWeb player. I'm not sure what the best option is here-

1. Remove field from playground (and maybe spec???).
2. Fix LottieWeb.
3. Use a different player (Skottie?).
4. Leave as is and document player issue/hope there is no confusion.

Reply:
IMO we should not include in specs 1.0 features that are not widely supported by existing players

Reply:
Are you thinking about this as reducing the scope for 1.0 specifically, or more of a general direction going forward?  Reducing 1.0 scope sounds reasonable, but if the latter, we should try to clarify a few points:

- isn't player capability/support supposed to be covered by profiles?
- what is the actual player set we are considering when making decisions?
- is the feature intersection going to be too restrictive?
- in practice, how do we determine support for players not actively represented?
- support vs implementation bugs - e.g. inverted masks appear to be supported in LottieWeb, they just produce incorrect results (in some cases/backends?)
- not all players are developed/supported at the same pace - what are the implications for long term format evolution?

For the issue at hand, I think I'm fine with dropping inverted masks (and mask expansion, and the uncommon mask modes) from 1.0, but IMO they should be added to the spec in future revisions even if not supported by all players.

Reply:
(Repeating here for visibility) I think just for 1.0 we should have features that are supported by most players and worry about profiles etc. after we published the first version of the specs

Reply:
Sounds good, thanks.  I dropped the inverted option, expansion, and the uncommon/unsupported mask modes.